src/igvfd/schemas/changelogs/technical_sample.md

```diff
@@ -3,3 +3,4 @@
 ### Changes since schema version 1
 
 * technical_sample has aliases mixin and submitted mixin removed, but still has properties for aliases and submitted due to change of basic_item mixin.  The actual properties have not changed.
+* Replaced date property with date_obtained which is inherited from sample.
```

Comment:
Even though we talk about inheritance, it is just mixing in properties from other schemas, so I would suggest avoiding saying inherited, instead, you can say "It is introduced in Technical sample schema through the mixin mechanism from Sample schema"